test/terraform_aws_s3_private_bucket_test.go

```diff
@@ -248,5 +255,44 @@ func TestTerraformAwsS3PrivateBucketCustomPolicy(t *testing.T) {
 
 	terraform.InitAndApply(t, terraformOptions)
 
+	// Check the bucket name
+	bucketName := terraform.Output(t, terraformOptions, "id")
+	assert.NotEmpty(t, bucketName)
+	assert.Equal(t, bucketName, testName)
+
 	aws.AssertS3BucketPolicyExists(t, awsRegion, testName)
 }
+
+func TestTerraformAwsS3PrivateBucketWithPrefix(t *testing.T) {
+	t.Parallel()
+
+	tempTestFolder := test_structure.CopyTerraformFolderToTemp(t, "../", "examples/prefix-bucket")
+	testName := fmt.Sprintf("terratest-aws-s3-private-bucket-%s", strings.ToLower(random.UniqueId()))
+	testPrefix := "custom"
```

Comment:
Since the CircleCI IAM restricts which buckets it can create/delete, the name will need to be something like https://github.com/trussworks/legendary-waddle/blob/master/trussworks-ci/admin-global/circleci.tf#L34